pkg/scheduler/nodeinfo/util_test.go

```diff
@@ -68,6 +68,30 @@ func TestGetNodeImageStates(t *testing.T) {
 				},
 			},
 		},
+		{
+			node: &v1.Node{
+				ObjectMeta: metav1.ObjectMeta{Name: "node-0"},
+				Status: v1.NodeStatus{
+					Images: []v1.ContainerImage{},
+				},
+			},
+			imageExistenceMap: map[string]sets.String{
+				"gcr.io/10:v1":  sets.NewString("node-0", "node-1"),
```

Comment:
Why is "node-0" in the `imageExistenceMap`? My understanding is that it shouldn't be in the map here and the other test case below.

Reply:
you're right, it should not be in the `imageExistenceMap`. Fixed.